app/src/main/java/org/fossasia/phimpme/accounts/AccountActivity.java

```diff
@@ -242,7 +238,7 @@ public void onClick(DialogInterface dialog, int which) {
     }
 
     private void signInDropbox() {
-        if (accountPresenter.checkAlreadyExist(accountName[7]))
+        if (accountPresenter.checkAlreadyExist(accountName[8]))
```

Comment:
@anantprsd5 Better idea is to store these integers as constants with proper variable names.